templates/ammos-cubs-ait.template.yaml

```diff
@@ -125,12 +219,12 @@ Resources:
           function exitTrap(){
           exitCode=$?
           /opt/aws/bin/cfn-signal \
-          --stack ${AWS::StackName} \
+          --stack ast-sandbox-main-AmmosCubsAITStack-1BQ7JZPVM3HIB \
           --resource AitAutoScalingGroup \
-          --region ${AWS::Region} -e $exitCode || echo 'Failed to send Cloudformation Signal'
+          --region us-gov-west-1 -e $exitCode || echo 'Failed to send Cloudformation Signal'
```

Comment:
🙅 no hard-coding

Reply:
Is there a reason you replaced this anyways? the `${AWS::X}` pseudo params should always be used and will make sure you can deploy more flexibibly